### ISSUE TITLE
6DoF Annotation 

### ISSUE THREAD

**Issue by wengdunfang** [zhanpenghe/6dof-annotation#10]
Hello, I am very happy to see your repo. I would like to ask if this is a 6DOF annotation script? How is it used? At the same time, I would like to ask you, do you have any recommended 6DoF annotation software? thank you.

**Comment by zhanpenghe** [comment]
Yes, this is a very simple 6dof annotation script with known mesh of the (rigid) object. I don't have any suggestion on these software and I wrote this before for another robotics project that I worked on. If the object is rigid and the object set is not humongous, this one should work well enough.
